R/estimateDominance.R

```diff
@@ -342,18 +342,21 @@ setMethod("estimateDominance", signature = c(x = "SummarizedExperiment"),
     ans
 }
 
-.get_dominance_values <- function(index, assay, ntaxa = 1, aggregate = TRUE) {
+.get_dominance_values <- function(index, mat, ntaxa = 1, aggregate = TRUE, ...) {
 
     FUN <- switch(index,
                   simpson_lambda = .simpson_lambda,
                   core_abundance = .calc_core_dominance,
                   gini = .calc_gini_dominance,
                   .calc_dominance)
```

Comment:
Named option?

Reply:
Do you refer to .calc_dominance here? This is a general option related to dbp, dmn, absolute and relative. They are all closely related with some differing parameterizations but also separately known by their specific names.

Reply:
The problem is, that if `index` is `whatever`, an index calculate as well. This might be just available from internal point of view, but also, when going through the code, it does not make sense with prior knowledge. Therefore, I would suggest explicitly naming each option.

Reply:
right. will do.